client/my-sites/checkout/upsell-nudge/plan-upgrade-upsell/index.jsx

```diff
@@ -87,103 +79,55 @@ export class PlanUpgradeUpsell extends PureComponent {
-								{
-									args: {
-										premiumThemePriceLow: formatCurrency( premiumThemePriceLow, currencyCode, {
-											precision: 0,
-										} ),
-										premiumThemePriceHigh: formatCurrency( premiumThemePriceHigh, currencyCode, {
-											precision: 0,
-										} ),
-									},
-									components: { b: <b /> },
-								}
-							) }
-						</p>
-						<p>
-							{ translate(
-								'But if you upgrade to a Premium plan with this special offer, you will get our full collection premium themes for just an additional %(discountPrice)s!',
-								{
-									args: {
-										discountPrice: formatCurrency( planDiscountedRawPrice, currencyCode, {
-											stripZeros: true,
-										} ),
-									},
-								}
+								'If you’re familiar with WordPress plugins, you know why that’s exciting. Installing plugins on your site is like downloading an app on your phone. Except, instead of getting a new game or productivity tool, you get new features and functionality for your site.'
 							) }
 						</p>
 						<p>
 							{ translate(
-								"You'll also gain access to some of the most powerful features on WordPress.com:"
+								'There are more than 50,000 WordPress plugins available to help turn your site into any powerful platform you imagine.'
 							) }
 						</p>
-
+						<p>{ translate( 'With the power of plugins you can:' ) }</p>
 						<ul className="plan-upgrade-upsell__checklist">
 							<li className="plan-upgrade-upsell__checklist-item">
 								<Gridicon icon="checkmark" className="plan-upgrade-upsell__checklist-item-icon" />
 								<span className="plan-upgrade-upsell__checklist-item-text">
 									{ translate(
-										'{{b}}More ways to monetize your site.{{/b}} You can sell stuff on your site without any hassle. Or earn through our special advertising program. Or why not both?',
+										'Transform your site into a mobile app, a job board, a wiki, or a coupon site.',
 										{
-											components: { b: <b /> },
 											comment: "This is a benefit listed on a 'Upgrade your plan' page",
 										}
 									) }
 								</span>
 							</li>
+							<li className="plan-upgrade-upsell__checklist-item">
+								<Gridicon icon="checkmark" className="plan-upgrade-upsell__checklist-item-icon" />
+								<span className="plan-upgrade-upsell__checklist-item-text">
+									{ translate( 'Create a private, customer-only section.', {
+										comment: "This is a benefit listed on a 'Upgrade your plan' page",
+									} ) }
+								</span>
+							</li>
+							<li className="plan-upgrade-upsell__checklist-item">
+								<Gridicon icon="checkmark" className="plan-upgrade-upsell__checklist-item-icon" />
+								<span className="plan-upgrade-upsell__checklist-item-text">
+									{ translate( 'Let your customers book and pay for appointments by themselves.', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( '<b>Appointments+:</b> Let your customers book and pay for appointments by themselves.' )` *ES Score: 8*